src/uwtools/drivers/shave.py

```diff
@@ -72,8 +70,8 @@ def _runcmd(self):
         """
         executable = self._driver_config["execution"]["executable"]
         config = self._driver_config["config"]
-        input_file = config.get("input_grid_file")
+        input_file = config["input_grid_file"]
         output_file = input_file.replace(".nc", "_NH0.nc")
-        flags = [config.get(key) for key in ["nx", "ny", "nh4", "input_grid_file"]]
+        flags = [config[key] for key in ["nx", "ny", "nh4", "input_grid_file"]]
```

Comment:
The lookups on these updated lines are for mandatory, schema-checked values, so I think `[]` should always be correct and better expresses the intent that these values _must` be present.